frontend/data/sponsorData.js

```diff
@@ -0,0 +1,573 @@
+    "appian_2021.jpg",
+    "GOLD",
+    appianDescription(),
+  ],
+  Amstelveen: [
+    "Amstelveen",
+    "https://amstelveen.com/",
+    "Amstelveen_notchecked.png",
+    "BRONZE",
+    amstelveenDescription(),
+  ],
+  Blackbird: [
+    "Blackbird",
+    "https://blackbird.vc/",
+    "blackbird.png",
+    "GOLD",
+    blackbirdDescription(),
+  ],
+  CISCO: [
+    "CISCO",
+    "https://www.cisco.com/c/en_au/index.html",
+    "CISCO-diamond.png",
+    "DIAMOND",
+    placeHolderDescription(),
+  ],
+  Commonwealth: [
+    "Commonwealth Bank (CBA)",
+    "https://www.commbank.com.au/about-us/careers.html",
+    "cba.jpg",
+    "SILVER",
+    placeHolderDescription(),
+  ],
+  EY: ["EY", "https://www.ey.com/en_au", "ey.png", "SILVER", eyDescription()],
+  Fdm: [
+    "FDM Group",
+    "https://www.fdmgroup.com/en-au/au-home/",
+    "fdm-group.png",
+    "SILVER",
+    fdmDescription(),
+  ],
+  Google: [
+    "Google",
+    "https://careers.google.com/",
+    "google-major.png",
+    "GOLD",
+    googleDescription(),
+  ],
+  Honeywell: [
+    "Honeywell,",
+    "https://www.honeywell.com/us/en",
+    "honeywell.png",
+    "GOLD",
+    placeHolderDescription(),
+  ],
+  IMC: ["IMC", "https://www.imc.com/ap/", "IMC.png", "GOLD", imcDescription()],
+  JaneStreet: [
+    "JaneStreet",
+    "https://www.janestreet.com/",
+    "jane-street.png",
+    "GOLD",
+    janeStreetDescription(),
+  ],
+  Macquarie: [
+    "Microsoft",
+    "https://www.macquarie.com.au/",
+    "macquarie.png",
+    "GOLD",
+    placeHolderDescription(),
+  ],
+  MantelGroup: [
+    "Mantel Group",
+    "https://mantelgroup.com.au/",
+    "mantel-group.png",
+    "SILVER",
+    mantelGroupDescription(),
+  ],
+  McGrathNicol: [
+    "McGrathNicol",
+    "https://www.mcgrathnicol.com/",
+    "mcgrathnicol-major.png",
+    "GOLD",
+    mcGrathNicolDescription(),
+  ],
+  McKinsey: [
+    "McKinsey",
+    "https://www.mckinsey.com/",
+    "mckinsey.png",
+    "SILVER",
+    placeHolderDescription(),
+  ],
+  Microsoft: [
+    "Microsoft",
+    "https://www.microsoft.com/en-au",
+    "microsoft_notchecked.png",
+    "GOLD",
+    placeHolderDescription(),
+  ],
+  Nakatomi: [
+    "Nakatomi",
+    "https://nakatomi.com/",
+    "Nakatomi.png",
+    "DIAMOND",
+    nakatomiDescription(),
+  ],
+  Palantir: [
+    "Palantir",
+    "https://www.palantir.com/",
+    "palantir-support.png",
+    "GOLD",
+    palantirDescription(),
+  ],
+  Pwc: [
+    "Pwc",
+    "https://www.pwc.com.au/",
+    "pwc-logo-png-transparent.png",
+    "SILVER",
+    pwcDescription(),
+  ],
+  Quantium: [
+    "Quantium",
+    "https://quantium.com/",
+    "quantium-major.png",
+    "GOLD",
+    quantiumDescription(),
+  ],
+  Telstra: [
+    "Telstra",
+    "https://www.telstra.com.au/",
+    "telstra.png",
+    "GOLD",
+    placeHolderDescription(),
+  ],
+  WooliesX: [
+    "WooliesX",
+    "https://wooliesx.com.au/",
+    "WooliesX.png",
+    "GOLD",
+    placeHolderDescription(),
+  ],
```

Comment:
Add logos and descriptions for VMWare and VivCourt

Reply:
there are no descriptions in the sponsor doc for VMWare or VivCourt, so I've added placeholder ones